components/VC/common/VCUtils.tsx

```diff
@@ -248,65 +257,106 @@ export const getMosipLogo = () => {
 /**
  *
  * @param wellknown (either supportedCredential's wellknown or whole well known response of issuer)
- * @param idType
- * @returns id Type translations (Eg - National ID)
+ * @param credentialConfigurationId
+ * @returns credential type translations (Eg - National ID)
  *
- * supportedCredential's wellknown is passed from getActivityText after fresh download
+ * supportedCredential's wellknown is passed from getActivityText after fresh download and viewing of card's mini view flow
```

Comment:
Can we update this comment with caller flows - mini view & detail view or remove this point?